imio/dms/mail/tests/test_subscribers.py

```
@@ -167,6 +171,202 @@ def test_dmsincomingmail_transition(self):
+        # C3: file with cleaned_download=True → skip
+        IAnnotations(afile)["documentgenerator"]["need_mailing"] = False
+        IAnnotations(afile)["documentgenerator"]["cleaned_download"] = True
+        with patch("imio.dms.mail.subscribers.convert_odt") as mock_conv:
+            dmsoutgoingmail_transition(omail, make_event("mark_as_sent"))
+            mock_conv.assert_not_called()
+
+        # C4: eligible file → convert_odt called, cleaned_download set to True
+        doc_annot = IAnnotations(afile)["documentgenerator"]
+        doc_annot["cleaned_download"] = False
+        with patch("imio.dms.mail.subscribers.convert_odt", return_value=b"new_content") as mock_conv:
+            with patch("imio.dms.mail.subscribers.getMultiAdapter") as mock_adapter:
```

Ruff (0.15.1)

[error] Local variable `mock_adapter` is assigned to but never used

Remove assignment to unused variable `mock_adapter`

(F841)